src/v1.js

```diff
@@ -20,6 +20,7 @@ const roleToJson = (role, context) => _.defaults(
 /** API end-point for version v1/ */
 var api = new API({
   title:      'Authentication API',
+  name:       'auth',
```

Comment:
Move to a separate PR, which I suspect we already have.

Reply:
yes it is merged on other PR. will remove it.